packages/headless/src/entity/user.ts

```diff
@@ -0,0 +1,13 @@
+export class UserEntity {
+  _id?: string;
```

Comment:
We won't use _id so far. because this is for mongodb only. instead we want to keep id: string as unique identifier